agent/plugins/spring-plugin/src/main/resources/META-INF/glowroot.plugin.json

```diff
@@ -47,6 +47,18 @@
       "transactionNameTemplate": "Spring scheduled: {{this.class.simpleName}}#{{methodName}}",
       "traceEntryMessageTemplate": "Spring scheduled: {{this.class.name}}.{{methodName}}()",
       "timerName": "spring scheduled"
+    },
+    {
+      "classAnnotation": "org.springframework.stereotype.Component|org.springframework.stereotype.Service",
+      "methodAnnotation": "org.springframework.amqp.rabbit.annotation.RabbitListener",
+      "methodParameterTypes": [
+        ".."
+      ],
+      "captureKind": "transaction",
+      "transactionType": "Amqp",
```

Comment:
Hi! In general this looks good. I think the `transactionType` here should probably be `Background` to be consistent with other queue processing instrumentation, e.g. https://github.com/glowroot/glowroot/blob/v0.13.4/agent/plugins/spring-plugin/src/main/resources/META-INF/glowroot.plugin.json#L34 and https://github.com/glowroot/glowroot/blob/v0.13.4/agent/plugins/jms-plugin/src/main/resources/META-INF/glowroot.plugin.json#L14